src/schema/comments.ts

```diff
@@ -731,9 +731,15 @@ export const resolvers: IResolvers<any, Context> = {
       await ctx.con.transaction(async (entityManager) => {
         const repo = entityManager.getRepository(Comment);
         const comment = await repo.findOneByOrFail({ id });
```

Comment:
When using TypeORM, you can indicate that you are going to use the `post` relation to eagerly fetch it, instead of lazy load it later. I think you should do it to optimize. You need to use `findOneOrFail` without the `by`. And provide a `relations` array in the object

Reply:
Ah cool yes, let me refactor it.